content/open-source/sigstore/policy-controller/how-to-install-policy-controller.md

```diff
@@ -0,0 +1,218 @@
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "sigstore" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+Now install the Policy Controller into the `cosign-system` namespace that you created in the first step of this guide:
+
+```bash
+helm install policy-controller -n cosign-system sigstore/policy-controller --devel
+```
+
+The `--devel` flag will include any alpha, beta, or release candidate versions of a chart. You can specify a particular version with the `--version` flag if you prefer.
+
+It may take a few minutes for your cluster to deploy all of the manifests needed for the Policy Controller. Check the status of your cluster using the `kubectl wait` command like this:
+
+```bash
+kubectl -n cosign-system wait --for=condition=Available deployment/policy-controller-webhook && \
+kubectl -n cosign-system wait --for=condition=Available deployment/policy-controller-policy-webhook
+```
+
+Once the Policy Controller deployments are done you will receive output like the following:
+
+```
+deployment.apps/policy-controller-webhook condition met
+deployment.apps/policy-controller-policy-webhook condition met
+```
+
+A full list of the resources that the Policy Controller deploys into your cluster is available at the end of this guide in [Appendix — Resource Types](#appendix--resource-types).
+
+You have now deployed the Policy Controller into your cluster. The next step is to enable it for the namespaces that you want to enforce policies in.
+
+## Step 3 — Enabling the Policy Controller
+
+Now that you have the Policy Controller installed into your cluster, the next step is to decide which namespaces should use it. By default, namespaces must enroll into enforcement, so you will need to label any namespace that you will use with the Policy Controller.
+
+Run the following command to include the `default` namespace in image validation and policy enforcement:
+
+```bash
+kubectl label namespace default policy.sigstore.dev/include=true
+```
+
+Apply the same label to any other namespace that you want to use with the Policy Controller.
+
+Now you can test enforcement by running a sample pod:
+
+```bash
+kubectl run --image cgr.dev/chainguard/nginx:latest nginx
+```
+
+The Policy Controller will deny the admission request with a message like the following:
+
+```
+Error from server (BadRequest): admission webhook "policy.sigstore.dev" denied the request: validation failed: no matching policies: spec.containers[0].image
+cgr.dev/chainguard/nginx@sha256:628a01724b84d7db2dc3866f645708c25fab8cce30b98d3e5b76696291d65c4a
+```
+
+The image is not admitted into the cluster because there are no `ClusterImagePolicy` (CIP) definitions that match it. Save and apply the following policy that will allow any [Chainguard Image](https://www.chainguard.dev/chainguard-images) hosted on the `cgr.dev/chainguard` registry to run on a cluster, while denying any other images:
```

Comment:
@ltagliaferri how's this section, from line 125-176 look for an explanation? Should I add a section heading?